.github/workflows/reusable-tsan.yml

```
@@ -41,7 +45,7 @@ jobs:
         sudo sysctl -w vm.mmap_rnd_bits=28
     - name: TSAN Option Setup
       run: |
-        echo "TSAN_OPTIONS=suppressions=${GITHUB_WORKSPACE}/${{ inputs.suppressions_path }}" >> $GITHUB_ENV
+        echo "TSAN_OPTIONS=log_path=${GITHUB_WORKSPACE}/tsan_log suppressions=${GITHUB_WORKSPACE}/${{ inputs.suppressions_path }} handle_segv=0" >> $GITHUB_ENV
```

Should this use the `tsan_logs_artifact_name` variable?

```suggestion
        echo "TSAN_OPTIONS=log_path=${GITHUB_WORKSPACE}/${{ inputs.tsan_logs_artifact_name }} suppressions=${GITHUB_WORKSPACE}/${{ inputs.suppressions_path }} handle_segv=0" >> $GITHUB_ENV
```

---

See comment above.